rest-server/src/main/java/io/github/nubesgen/configuration/NubesgenConfiguration.java

```diff
@@ -193,6 +193,8 @@ public boolean isRuntimeDocker() {
         return (
```

Comment:
should not this be better as a method of `RuntimeType`. e.g. `RuntimeType::isDocker?`